modules/tensor_mechanics/src/materials/RadialReturnStressUpdate.C

```diff
@@ -12,7 +12,7 @@ template <>
 InputParameters
 validParams<RadialReturnStressUpdate>()
 {
-  InputParameters params = validParams<StressUpdateBase>();
+  InputParameters params = validParams<StateUpdateBase>();
   params.addClassDescription("Calculates the effective inelastic strain increment required to "
                              "return the isotropic stress state to a J2 yield surface.  This class "
                              "is intended to be a parent class for classes with specific "
```

Comment:
Would you add in a check with Daniels' isotropic tensor guarantee to this class?  Or, since it is very much scope creep, feel free to tag me to complete this check here :-)